snaps/learn/tutorials/gas-estimation.md

```diff
@@ -28,103 +28,146 @@ starter kit by running:
 yarn create @metamask/snap gas-estimation-snap
 ```
 
-or
-
-```bash
-npx @metamask/create-snap gas-estimation-snap
-```
-
-or
-
-```bash
-npm create @metamask/snap gas-estimation-snap
-```
-
 Next, `cd` into the `gas-estimation-snap` project directory and run:
 
 ```bash
 yarn install
 ```
 
-This initializes your development environment with the required dependencies.
-You may get a warning similar to the following:
+This initializes your development environment with the required dependencies. 
+
+<details>
+  <summary>Did you get a warning?</summary>
+  <div>
+You may get a warning such as: 
 
 ```bash
 @lavamoat/allow-scripts has detected dependencies without configuration. explicit configuration required.
 run "allow-scripts auto" to automatically populate the configuration.
 ```
-
-You can resolve this error by running the following command:
-
+You can resolve this error by running:
+    
 ```bash
 yarn run allow-scripts auto
 ```
+ </div>
+</details>
+
+
+### 2. (Optional) customize your Snap
+
+This Snap is generated from a boilerplate typescript example Snap. While the following steps in the 
+drop-down are optional for testing, they significantly improve the UX by aligning the Snap with its function.
+
+<details>
+  <summary>Customize the Snap's UX</summary>
+  <div>
```

Comment:
```suggestion
This Snap is generated from a boilerplate TypeScript example Snap.
We recommend customizing your Snap to improve its UX, but this is optional for testing.
If you don't wish to customize your Snap, skip to [Step 3](#3-enable-network-access).

```

In my opinion, we should take the optional sub-steps out of the dropdown and make them into H4 headings (as Joan suggested) so they are visible in the right-hand TOC.

Having such a large amount of content in a dropdown may become cumbersome for users, and looks overly complex. And if we recommend doing the steps anyway, we should keep them visible.

My suggestion also adds a reference to Step 3 if the user wants to skip.